bedrock/base/templates/includes/banners/pocket-pencil-banner.html

```diff
@@ -0,0 +1,27 @@
+{#
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% if LANG == 'de' %}
+  {% set heading_text = 'Entdecke die besten <br class="mobile-break"> Artikel aus 2023'|safe %}
+  {% set cta_text = 'Zum Best of 2023' %}
+{% else %}
+  {% set heading_text = 'Discover the top <br class="mobile-break"> articles of 2023'|safe %}
+  {% set cta_text = 'See the winners' %}
+{% endif %}
+
+<aside class="pocket-banner c-banner hide-from-legacy-ie" id="pocket-banner">
```

Comment:
The right-most image overflows here on smaller screens, but I think this can be offset when you make the banner full-width as we discussed 👍🏼 

<kbd>![Screenshot 2024-01-04 at 11 15 32 AM](https://github.com/mozilla/bedrock/assets/42309026/dc654d19-4ce1-4c24-a41d-d35955a5a059)</kbd>